code/__DEFINES/is_helpers.dm

```diff
@@ -129,7 +129,7 @@ GLOBAL_LIST_INIT(turfs_without_ground, typecacheof(list(
 
 #define isregalrat(A) (istype(A, /mob/living/simple_animal/hostile/regalrat))
 
-#define isswarmer(A) (istype(A, /mob/living/simple_animal/hostile/swarmer))
+#define isswarmer(A) ( istype(A, /mob/living/simple_animal/hostile/swarmer) || istype(A, /mob/living/simple_animal/hostile/swarmer/drone) )
```

Comment:
istype already includes subtypes. did you test this?

Reply:
I did not and instead I made a stupid assumption.